### PR TITLE
[IMP] l10n_ar: report name of document types 60/61

### PR DESCRIPTION
**Pull request by bruno-zanotti**
Add report name and document prefix on l10n_ar data for
the following documents:
 - (60) CUENTAS DE VENTA Y LIQUIDO PRODUCTO A
 - (61) CUENTAS DE VENTA Y LIQUIDO PRODUCTO B

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by vbe-odoo** [comment]
Hi @Bruno-Zanotti this LGTM!
Ready for technical review @jco-odoo 
Thanks!

cc' @ren-odoo

File: addons/l10n_ar/views/report_invoice.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
                         <span style="font-size: x-small;" t-esc="not pre_printed_report and document_legend or '&#160;'"/>
                     </span>
                 </div>
-                <div name="right-upper-side" class="col-5 text-right" t-if="not pre_printed_report">
+                <div name="right-upper-side" class="col-5 text-right" style="padding-left: 0px;" t-if="not pre_printed_report">
```
Comment (jco-odoo) [inline addons/l10n_ar/views/report_invoice.xml:19]: @Bruno-Zanotti What does this have to do with it?

File: addons/l10n_ar/__manifest__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Argentinian Accounting',
-    'version': "3.0",
+    'version': "3.1",
```
Comment (jco-odoo) [inline addons/l10n_ar/__manifest__.py:5]: no need to update this all the time.  It also gives fw-port problems for no reason

**Review by jco-odoo — COMMENTED** [review]
please check

File: addons/l10n_ar/__manifest__.py (hunk 2 above)
Comment (jjscarafia) [reply addons/l10n_ar/__manifest__.py:5]: Hi @jco-odoo If I'm not wrong this is needed so that odoo.sh instances update de module and get the fix / improvement. We also need this so that in our infrastructure the modules are also updated.
But if needed we can skip this change. Just let us know. Thanks!

File: addons/l10n_ar/views/report_invoice.xml (hunk 1 above)
Comment (jjscarafia) [reply addons/l10n_ar/views/report_invoice.xml:19]: This is because of the lenght of the new document type.

Without this change:
[LP-B 00001-00000003.pdf](https://github.com/odoo/odoo/files/6067972/LP-B.00001-00000003.pdf)

With this change:
[LP-A 00001-00000002.pdf](https://github.com/odoo/odoo/files/6067976/LP-A.00001-00000002.pdf)

Comment (jco-odoo) [reply addons/l10n_ar/views/report_invoice.xml:19]: ok, makes sense

**Comment by jco-odoo** [comment]
@robodoo r+

File: addons/l10n_ar/__manifest__.py (hunk 2 above)
Comment (jco-odoo) [reply addons/l10n_ar/__manifest__.py:5]: It is updated a bit too much this way.  But let's continue.
